app/Models/WaitingRoom.php

```diff
@@ -15,4 +15,62 @@ public function users()
     {
         return $this->belongsToMany(User::class);
     }
+
+    /*
+     * Equally splits the users of a waiting room into arrays
+     * representing competitions.
+     */
+    public function getDefaultSplit()
+    {
+        $users = $this->users;
+
+        // Get the size of the waiting room
+        $roomSize = count($users);
+
+        // Determine the amount of competitions to make
+        $numOfCompetitions = $roomSize / 300;
```

Comment:
If the roomsize is a decimal nothing breaks,

and no we would never get a room with 5 people. Its always an even split (so 305.5 would have a 152 and 153 group or whatever the math is)